src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int RANDOM_FIGURE_PARAMETER = 100;
+    private static final int DEFAULT_FIGURE_RADIUS = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomFigureNum = random.nextInt(FIGURE_COUNT);
+        String color = colorSupplier.getRandomColor();
+        int firstRandomParameter = random.nextInt(RANDOM_FIGURE_PARAMETER);
```

Comment:
Create separate method for getting random side`random.nextInt(RANDOM_FIGURE_PARAMETER)` and use it for each type of figure